app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
When you start to style your app, don't forget about the Devise pages, you want to make sure they look like they belong with the rest of your app!